birdhouse/config/proxy/conf.d/all-services.include.template

```diff
@@ -4,17 +4,20 @@
 
     location /components/ {
         default_type application/json;
-        return 200 '${BIRDHOUSE_DEPLOY_COMPONENTS_JSON}';
+        root /static;
+        try_files /components.json =404;
     }
 
     location /services {
         default_type application/json;
-        return 200 '${BIRDHOUSE_DEPLOY_SERVICES_JSON}';
+        root /static;
+        try_files /services.json =404;
     }
 
     location /version {
```

Comment:
Does these endpoints properly work by themselves? 
i.e.: calling `https://${PAVICS_FQDN_PUBLIC}/services` and `https://${PAVICS_FQDN_PUBLIC}/version` without trailing `/` does not cause a redirect to `/jupyter/hub`?

Asking because `/components` does cause the redirect to Jupyter if not defined explicitly as `/components/`, due to how the location is defined above. If it works, maybe `/components/` should be changed to `/components` instead, to align all the endpoints together.

Reply:
Yeah I'd prefer to change it to `/components` too. It can be annoying if you forget that final `/`. I can do that in this PR